packages/backend/API.md

```diff
@@ -18,18 +18,26 @@ import { Stack } from 'aws-cdk-lib';
 export { a }
 
 // @public
-export type Backend<T extends Record<string, ConstructFactory<ResourceProvider>>> = {
-    createStack: (name: string) => Stack;
-} & {
+export type Backend<T extends Record<string, ConstructFactory<ResourceProvider>>> = BackendHelpers & {
     [K in keyof T]: ReturnType<T[K]['getInstance']>['resources'];
 };
 
+// @public (undocumented)
+export type BackendHelpers = {

```

Comment:
```suggestion
export type BackendBase = {

```

or `BackendCore`/`BackendCoreInterface`/`BackendInterface`

or anything that says "this is base backend interface" (that is then dynamically extended by type magic).

Reply:
updated to `BackendBase`